contingency/contingency-api/src/main/java/com/powsybl/contingency/tasks/ShuntCompensatorTripping.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.contingency.tasks;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.*;
+
+/**
+ * @author Teofil Calin BANC <teofil-calin.banc at rte-france.com>
+ */
+public class ShuntCompensatorTripping extends AbstractInjectionTripping {
+
+    private final String shuntCompensatorId;
```

Comment:
See comment on abstract class: could be shared in that class instead.

Reply:
done